src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -23,4 +37,19 @@ private SearchFilterFiles() {}
 				.collect( Collectors.toList() ); //
 	}
 
+	public static List<File> getProjectFilterFiles() {
+		if ( !resolveFilterPath().toFile().exists() ) {
+			ProjectConfiguration.getInstance().ensureProjectConfigurationInitialized();
+		}
+		try ( Stream<Path> paths = Files.walk( resolveFilterPath() ) ) {
+			return paths.filter( Files::isRegularFile ) //
+					.filter( file -> file.toString().endsWith( ".filter" ) ) //
+					.map( Path::toFile ) //
+					.collect( Collectors.toList() ); //
+		} catch ( final Exception e ) {
+			log.info( "You can create your own filter files in the " + File.separator + ".retest" + File.separator
+					+ "filter folder" );
```

Comment:
You are checking that the directory exists and creating it if necessary. You should not wait for an exception to be thrown. This should be part of the documentation, not the code.